src/RecoveryServices/RecoveryServices.SiteRecovery/ReplicationProtectedItem/SetAzureRmRecoveryServicesAsrReplicationProtectedItem.cs

```diff
@@ -56,6 +57,8 @@ public class SetAzureRmRecoveryServicesAsrReplicationProtectedItem : SiteRecover
         ///     Gets or sets the NIC of the virtual machine for which this cmdlet sets the recovery network property.
         /// </summary>
         [Parameter]
+        [CmdletParameterBreakingChange("PrimaryNic", ChangeDescription = "Further this property will be used to set primary NIC for Replication ProtectedItem."
```

Comment:
You need to describe what the actual change is - how will a user need to update their scripts for thsi change?

Reply:
[CmdletParameterBreakingChange("PrimaryNic", ChangeDescription = "Further this property will be used to set primary NIC for Replication ProtectedItem."
            + "And UpdateNic will be used to update(selected) provided NIC",
            OldWay = "Set-AzRecoveryServicesAsrReplicationProtectedItem -ReplicationProtectedItem $RPI -PrimaryNic $NicId -RecoveryNetworkId $AzureNetworkID -RecoveryNicSubnetName $subnetName",
            NewWay = "[To update the details for selected NIC] Set-AzRecoveryServicesAsrReplicationProtectedItem -InputObject $RPI -UpdateNic $NicId -RecoveryNicSubnetName $subnetName"
            + "[To update the primary NIC to be used after failover] Set-AzRecoveryServicesAsrReplicationProtectedItem -InputObject $RPI -PrimaryNic $NicId")]

Is there the way to add line change in "NewWay"